practices/automateRepeatableTasks/readme.md

```diff
@@ -0,0 +1,65 @@
+# Automate Repeatable Tasks
+
+Humans are error prone; we are very bad at doing the same thing over and over. Machines, on the other hand, are very good at repeatable tasks! When working on a software project, we should strive to reduce any manual steps as much as possible. This will save time, reduce errors, but also add predictability to our software. "Works on my machine" is never an acceptable excuse! =)
+
+## The What and the Why
+
+### The What
+
+Automation of repeatable task is pretty self explanatory: the process of implementing repeatable tasks automatically. Repeatable task can be found in many aspects of work at Concord such as planning, development and deployment. Depending on the type of task it may require different methods for automation.
```

Comment:
The first sentence here feels like filler, and can probably be omitted entirely.
Also, per the rhetoric of Vampire Weekend my feelings on an Oxford comma are indifferent, we have generally used them for the other documents so probably a good idea for consistency's sake (which I do care about).

Reply:
I rephrased the first sentence a bit and added the much loved oxford comma. The first sentence is a bit of a placeholder for a definition of the topic, following consistency from the other files. I agree it is filler, but I think it is important for consistency with the other wiki pages. What do you think?

Reply:
rephrasing looks good, resolving